src/scss/components/heroSlide.scss

```diff
@@ -0,0 +1,249 @@
+  grid-column: 2/14;
+}
+.carousel .true .hero__img{
+  grid-column: 1/15;
+  max-height: 617px;
+  grid-row: 1/3;
+  
+}
+.hero__slide__container img{
+  width: 100%;
+  height: auto;
+}
+.carousel .false img{
+  margin-top: 40px;
+}
+
+.grabbing {
+  cursor: grabbing;
+}
+.carousel .hero__slide__container.false .info {
+  grid-column: 2/14;
+  width: auto;
+  justify-content: center;
+}
+.carousel .hero__slide__container.true .info {
+  grid-column: 2/14;
+  width: auto;
+  justify-content: center;
+  align-items: flex-end;
+  grid-row: 2;
+}
+
+.carousel .hero__slide__container .info h1{
+  width: 100%;
+  text-align: center;
+  font-size: 36px;
+  line-height: 36px;
+  margin: 0;
+}
+
+.carousel .hero__slide__container .info p{
+  width: 100%;
+  text-align: center;
+  font-size: 16px;
+  line-height: 22px;
+  margin-top: 16px;
+}
+.hero__button{
+  width: fit-content;
+  display: flex;
+  flex-direction: row;
+  height: 43px;
+  background-color: var(--electric-blue);
+  border-radius: 4px;
+  padding:12px;
+  font-size: 16px;
+  color: var(--white);
+  text-align: center;
+  margin-top: 16px;
+  margin-left: auto;
+  margin-right: auto;
+  margin-bottom: 43px;
+}
+.icon{
+  width: 20px;
+  height: 20px;
+  margin-left: 10px;
+  margin-top: 3px;
+}
+
+.carousel .true img{
+  z-index: 1;
+  width: auto;
+  height: 100vh;
+  margin-left: 0;
+}
+@supports(object-fit: cover){
+  .carousel .true img{
+      height: 100vh;
+      object-fit: cover;
+      object-position: center center;
+    }
+}
+
+.true{
+  color: var(--white);
+}
+.fa-internalPage{
+  display: none !important;
+}
+.hero__slider{
+  z-index: 2;
+  display: flex;
+  margin-bottom: 10px;
+  position: absolute;
+  align-items: flex-end;
+  justify-content: center;
+  width: 100%;
+  margin-top: -30px;
+}
+.hero__slider .hero__slider__dot{
+  background: var(--baby-blue);
+  border-radius: 50%;
+  cursor: pointer;
+  width: 8px;
+  height: 8px;
+  color: transparent;
+  margin-right: 4px;
+  margin-left: 4px;
+}
+.hero__slider .hero__slider__dot.active{
+  background: var(--mid-blue);
+}
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+// Large devices (desktops, 992px and up)
+@media (min-width: 992px) {
+
+.carousel .false .hero__img{
+  grid-column: 2/9;
+}
+.carousel .true .hero__img{
+  grid-column: 1/15;
+  max-height: 617px;
+  grid-row: 1/3;
+  
+}
+.hero__slide__container img{
+  width: 100%;
+  height: auto;
+}
+.carousel .false img{
+  margin-top: 40px;
+}
+
+.carousel .hero__slide__container.false .info {
+  grid-column: 9/14;
+  width: auto;
+  justify-content: left;
+  margin-top: auto;
+  margin-bottom: auto;
+}
+.carousel .hero__slide__container.true .info {
+  grid-column: 9/14;
+  width: auto;
+  justify-content: center;
+  align-items: center;
+  margin-bottom: auto;
+}
+
+.carousel .hero__slide__container .info h1{
+  text-align: left;
+  font-size: 64px;
+  line-height: 62px;
+}
+
+.carousel .hero__slide__container .info p{
```

Comment:
use the class that you assigned to the element `.hero__description` instead of the `p` tag